### ISSUE TITLE
Current installation process breaks bandagebd's transparency

### ISSUE THREAD

**Issue by youwereeatenbyalid** [bb010g/betterdiscordctl#37]
Pretty much what it says on the tin. BandagedBD has apparently been updated to use a new installation method that uses a new injector (I assume to inject BandagedBD's code into discord) and the current installation method for linux doesn't follow that, causing a few things to break like the transparency option.

**Comment by rauenzi** [comment]
This injection method was implemented long ago in BBD by the other two platforms. I informed @bb010g of the changes last year and gave him exact steps needed for the new method.

Just in case you have time to update the installer @ObserverOfTime (don't worry this process is super easy) here they are:

1. Locate Discord binary directory (e.g. on Windows `%localappdata%\Discord\app-0.0.301`).
2. Enter the `resources` dir and create a folder named `app` if it doesn't exist.
3. Download the `injector` branch from the repo. (`https://github.com/rauenzi/BetterDiscordApp/archive/injector.zip` (or `.tar.gz`) works great)
4. Extract the contents of the branch to inside the `app` folder.
5. Restart Discord
